variantlib/models/provider.py

```diff
@@ -82,3 +87,9 @@ def pretty_print(self) -> str:
             )
         result_str += f"\n{'#' * 80}\n"
         return result_str
+
+    def to_list_of_properties(self) -> Generator[VariantProperty]:
```

Comment:
```suggestion
    @property
    def sorted_properties(self) -> list[VariantProperty]:
```

it doesn't need to be a generator. We are not talking about 10k objects.
This can easily be made a (cached?) property